pythonosc/osc_server.py

```diff
@@ -29,6 +39,42 @@
 from pythonosc import osc_packet
 
 
+def _handle(data, dispatcher):
```

Comment:
We need a better name for this method now that it is shared between different kind of classes, how about _call_handlers_for_packet ?